File: TAREA_7_U2/TAREA_7_U2/src/tarea_7_u2/TAREA_7_U2.java
package tarea_7_u2;

import java.util.Scanner;

public class TAREA_7_U2 {
       static Bank bank = new Bank();
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        Employee employee = new Employee(" ", " ", 132, ' ');
        Boolean flag=true;
        while (flag) {
            System.out.println("===============================================================");
            System.out.println("1. Add Acoount of Employee");
            System.out.println("2. Add Money");
            System.out.println("3. Withdrawals");
            System.out.println("4. Show Account of Usser");
            System.out.println("5. Show all Employees");
            System.out.println("6. Show all Accounts");
            System.out.println("7. Exit");
            System.out.println("Select your option: ");
            System.out.println("===============================================================");
            char option = sc.next().charAt(0);
            switch (option) {
                case '1':
                    employee.addEmployeeAccount();
                    break;
                case '2':
                    employee.addMoney();
                    break;
                case '3':
                    employee.withdrawMoney();
                    break;
                case '4':
                    employee.showAccount();
                    break;
                case '5':
                    employee.showAllEmployees();
                    break;
                case '6':
                    bank.showAllBankAccounts();
                    break;
                case '7':
                    System.out.println("Finishing all Tasks.");
                    flag = false;
                    break;
                default:
                    System.out.println("Not an option. Enter a number from 1 to 4.");
                    break;
            }
}}}

    

/*
public class Main {
    public static void main(String[] args) {

        Employee employee = new Employee("Juan", "Rivera", 101010, 'A');

        System.out.println(employee.getName());
        System.out.println(employee.getLastName());
        System.out.println(employee.getAccount().getAccountNumber());
        System.out.println(employee.getAccount().getType());
        System.out.println(employee.getAccount().getAmount());

        System.out.println("*********");

        employee.getAccount().agregarDinero(10000);
        System.out.println(employee.getAccount().getAmount());

        System.out.println("*********");

        employee.getAccount().agregarDinero(50000);
        System.out.println(employee.getAccount().getAmount());
    }
}
*/

File: Tarea_7_U2.py/main/Tarea_7_U2_.py
from main.Bank import Bank
from main.Employee import Employee


bank = Bank()

def main():
    flag = True
    while flag:
        print("===============================================================")
        print("1. Add Account of Employee")
        print("2. Add Money")
        print("3. Withdrawals")
        print("4. Show Account of User")
        print("5. Show all Employees")
        print("6. Show all Accounts")
        print("7. Exit")
        print("Select your option: ")
        print("===============================================================")
        option = input()
        if option == '1':
            employee.add_employee_account()
        elif option == '2':
            employee.add_money()
        elif option == '3':
            employee.withdraw_money()
        elif option == '4':
            employee.show_account()
        elif option == '5':
            employee.show_all_employees()
        elif option == '6':
            bank.show_all_bank_accounts()
        elif option == '7':
            print("Finishing all Tasks.")
            flag = False
        else:
            print("Not an option. Enter a number from 1 to 7.")
if __name__ == "__main__":
    employee = Employee("", "", 132, ' ')
    main()
